servers/src/main/java/tachyon/worker/block/evictor/EvictorUtils.java

```diff
@@ -26,6 +26,52 @@
 public final class EvictorUtils {
 
   /**
+   * Get StorageDirView with max free space.
+   *
+   * @param bytesToBeAvailable space size to be requested
+   * @param location location that the space will be allocated in
+   * @return the StorageDirView selected
+   */
+  public static StorageDirView getDirWithMaxFreeSpace(long bytesToBeAvailable,
```

Comment:
Since we have `EvictorBase` now, does it make sense to move the logic in `EvictorUtils` to `EvictorBase`? Not something this PR needs to address.